sherpa/conftest.py

```diff
@@ -47,13 +47,41 @@
     has_xspec = False
 
 
-import sherpa.plot
-import sherpa.astro.plot
-from sherpa.plot import dummy_backend
+from sherpa.plot.backends import BaseBackend, IndepOnlyBackend, PLOT_BACKENDS
+from sherpa.plot import TemporaryPlottingBackend
+
+FUNCTIONAL_PLOT_BACKENDS = []
+'''Unfortunately, there is no metadata that tell us which plotting backends are
+dummys and which ones actually produce useful plots. That's because
+"one man's dummy might be another man's usecase", e.g. a backend might be a
```

Comment:
and while reading other people's suggestions, I realized that my wording in this comment is inappropriate and might be offensive to non-male developers; I apologize.
```suggestion
"one person's dummy might be another person's usecase", e.g. a backend might be a
```